charts/victoria-metrics-k8s-stack/templates/_helpers.tpl

```diff
@@ -297,6 +297,12 @@
   {{- $Chart := (.helm).Chart | default .Chart }}
   {{- $selectSpec := include "vm.select.spec" . | fromYaml -}}
   {{- $clusterSpec := deepCopy $Values.vmcluster.spec -}}
+  {{- $replicationFactor := (int $clusterSpec.replicationFactor) | default 1 }}
+  {{- $storageNodes := (int ($clusterSpec.vmstorage).replicaCount) | default 1 }}
+  {{- $minStorageNodes := sub (mul 2 $replicationFactor) 1 }}
+  {{- if gt $minStorageNodes $storageNodes }}
+    {{ fail (printf "storage nodes count %d is smaller than expected %d" $storageNodes $minStorageNodes) }}
```

Comment:
Not sure if chart should force this, it's the rule for maintain the given replication factor for newly ingested data when N-1 of storage nodes are unavailable. Maybe user don't want to follow that:)

But yeah, the change should be good.
Could you add a update note to the change log?

Reply:
done